nbgrader/apps/autogradeapp.py

```diff
@@ -35,76 +31,96 @@ class AutogradeApp(BaseNbConvertApp):
-        name and the student id):
-        
-        > nbgrader autograde "Problem 1.ipynb" --assignment="Problem Set 1" --student=student1
-
-        If you want to override the .nbconvert part of the filename, you can use
-        the --output flag:
+        Autograde submitted assignments. This takes one argument for the
+        assignment id, and then (by default) autogrades assignments from the
+        following directory structure:
 
-        > nbgrader autograde "Problem 1.ipynb" --output "Problem 1.graded.ipynb" --assignment="Problem Set 1" --student=student1
+            submitted/*/{assignment_id}/*.ipynb
 
-        Or, you can put the graded version in a different directory. In the
-        following example, there will be a file "graded/Problem 1.ipynb" after
-        running `nbgrader autograde`:
+        and saves the autograded files to the corresponding directory in:
 
-        > nbgrader autograde "Problem 1.ipynb" --build-dir=graded --assignment="Problem Set 1" --student=student1
+            autograded/{student_id}/{assignment_id}/{notebook_id}.ipynb
 
-        You can also use shell globs, and copy files from one directory to another:
+        The student IDs must already exist in the database. If they do not, you
+        can tell `nbgrader autograde` to add them on the fly by passing the
+        --create flag.
 
-        > nbgrader autograde submitted/*.ipynb --build-dir=graded --assignment="Problem Set 1" --student=student1
+        Note that the assignment must also be present in the database. If it is
+        not, you should first create it using `nbgrader assign`. Then, during
+        autograding, the cells that contain tests for the students' answers will
+        be overwritten with the master version of the tests that is saved in the
+        database (this prevents students from modifying the tests in order to
+        improve their score).
 
-        If you want to overwrite grade cells with the source and metadata that
-        was stored in the database when running `nbgrader assign` with --save-cells,
-        you can use the --overwrite-cells flag:
+        To grade all submissions for "Problem Set 1":
+            nbgrader autograde "Problem Set 1"
 
-        > nbgrader autograde "Problem 1.ipynb" --assignment="Problem Set 1" --student=student1 --overwrite-cells
+        To grade only the submission by student with ID 'Hacker':
+            nbgrader autograde "Problem Set 1" --student Hacker
 
+        To grade only the notebooks that start with '1':
+            nbgrader autograde "Problem Set 1" --notebook "1*"
         """
     ))
 
-    overwrite_cells = Bool(
-        False, 
-        config=True, 
+    create_student = Bool(
+        False, config=True,
+        help=dedent(
+            """
+            Whether to create the student at runtime if it does not
+            already exist.
+            """
+        )
+    )
+
+    nbgrader_step_input = Unicode(
+        "submitted",
```

Comment:
Yeah, I'm not a big fan of the name either. How about `nbgrader_input_step_name`?